cloud/helm/aws/charts/backup-restore/templates/_scripts.tpl

```diff
@@ -0,0 +1,45 @@
+{{- define "liferayAWSBackupRestore.script.getDependenciesModuleOutputs" -}}
+{{- $terraformOutputDBRestoreSnapshotIdentifier := "db_restore_snapshot_identifier" -}}
+{{- $terraformOutputIsRestoring := "is_restoring" -}}
+{{- include "liferayAWSBackupRestore.script.partial.shStart" . }}
+
+function main {
+    local db_restore_snapshot_identifier
+
+    terraform init -input=false
+
+    terraform plan -detailed-exitcode -input=false
+
+    db_restore_snapshot_identifier=$( \
+        terraform \
+            output \
+            -raw \
+            {{ $terraformOutputDBRestoreSnapshotIdentifier }} 2>/dev/null || echo "")
```

Comment:
Terraform output does not show an output with values that are equal to null, so catch the error.

Not relevant for the other output since it's a boolean.